themes/theme-b2b/src/components/Popover.styles.ts

```diff
@@ -1,12 +1,15 @@
 import { ThemeComponent, cva } from '@marigold/system';
 
-export const Popover: ThemeComponent<'Popover'> = cva(['mt-0.5'], {
```

Comment:
I don't see a Popover component in Storybook—where do I find this?

Reply:
You can see this for example in the `DatePicker` when you open the `Calendar`, or in the `Select` component. here is an example: 
<img width="1688" alt="Bildschirmfoto 2024-01-17 um 09 03 41" src="https://github.com/marigold-ui/marigold/assets/38324334/8d652b0a-888b-4f81-aae5-a565e04b6776">

Reply:
Thanks Sarah, I found it. In both the Date Picker and Select components, these lines of code have an additional class name at the end: "w-[--trigger-width]". Does that matter?
![Captura de pantalla 2024-01-17 a la(s) 11 35 30](https://github.com/marigold-ui/marigold/assets/156671137/d84e4aac-9ab6-48ab-83c7-7c4da1c3acdf)